fontforge/scripting.c

```diff
@@ -3278,9 +3281,8 @@ static void bLoadTableFromFile(Context *c) {
 	sf->ttf_tab_saved = tab;
     } else
 	free(tab->data);
-    tab->len = len;
```

Comment:
It seems the `len` variable is unused after your patch, could be cleaned up completely.